tests/environment-setup/test_used_docker_compose.py

```diff
@@ -0,0 +1,61 @@
+# pylint: disable=unused-argument
+# pylint: disable=bare-except
+# pylint: disable=redefined-outer-name
+
+import re
+from pathlib import Path
+from typing import List, Tuple
+
+import pytest
+
+
+@pytest.fixture(scope="module")
+def docker_compose_in_requirement_files(
+    osparc_simcore_root_dir: Path,
+) -> List[Tuple[Path, str]]:
+
+    reqs_path_version_tuples = []
+
+    for req_path in osparc_simcore_root_dir.rglob("*.txt"):
+        found = re.search(r"docker-compose==([\d\.]+)", req_path.read_text())
+        if found:
+            version = found.group(1)
+            reqs_path_version_tuples.append((req_path, version))
+
+    return reqs_path_version_tuples
+
+
+@pytest.fixture(scope="module")
+def docker_compose_in_ci_script(osparc_simcore_root_dir: Path) -> Tuple[Path, str]:
+    setup_ci = osparc_simcore_root_dir / "ci/github/helpers/setup_docker_compose.bash"
+
+    found = re.search(r"DOCKER_COMPOSE_VERSION=\"([\d\.]+)\"", setup_ci.read_text())
+    assert found, f"Expected DOCKER_COMPOSE_VERSION=x.x.x in {setup_ci}"
+
+    version = found.group(1)
+    return (setup_ci, version)
+
+
+def test_all_docker_compose_have_same_version(
+    docker_compose_in_requirement_files,
+    docker_compose_in_ci_script,
+    osparc_simcore_root_dir,
+):
+    previous = docker_compose_in_requirement_files[0]
```

Comment:
just wondering if we should not prevent it from happening?

Reply:
I agree but I think this is already a good start.

Enforcing the same version of docker-compose everywhere would still imply some work:
- making sure "docker-compose" is never installed in any service/package  
    - remove pytest-docker everywhere (I would prefer to do this in a different PR)
    - still check that no other library creates the dependency (i.e. this test)
- create recipe in main Makefile that sets up the developer host correctly (e.g. installs docker, docker-compose, python etc). Most of these scripts we already have them under ``ci``
- ensure that setups in CI setups and developer host are the same

Reply:
in short. I am for it, but I rather do a new PR with it, if you do not mind :-)

Reply:
and one question. why we do not update to 1.27.2? let's do that in a next PR.